x/gamm/keeper/keeper.go

```diff
@@ -70,14 +65,3 @@ func (k *Keeper) SetHooks(gh types.GammHooks) *Keeper {
 
 	return k
 }
-
-// GetParams returns the total set params.
-func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
-	k.paramSpace.GetParamSet(ctx, &params)
-	return params
-}
-
-// SetParams sets the total set of params.
-func (k Keeper) setParams(ctx sdk.Context, params types.Params) {
-	k.paramSpace.SetParamSet(ctx, &params)
-}
```

Comment:
should we create an issue to come back and migrate the params?

Reply:
What do you mean by migrate? They are completely removed. I don't see any logic to explicitly clear param state in the `x/params` module

Reply:
If you mean migrate to swaprouter, then the only parameter is set in the upgrade handler